SpatialGDK/Source/SpatialGDK/Public/Utils/LatencyManager.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+#include <WorkerSDK/improbable/c_worker.h>
+
+#include "LatencyManager.generated.h"
+
+class USpatialNetConnection;
+class USpatialNetDriver;
+
+UCLASS(transient)
```

Comment:
Why does this need to be transient?

Reply:
Copy pasta error, it shouldn't need to be transient